### PR TITLE
Update committee lists

### LINKED ISSUE #1556: Updating Committees Participant lists
Can you please update 
**https://jakarta.ee/committees/marketing/**, Committee Members list with (the change is in bold text here; on the website no need to need to bold)

Kenji Kazumura - Fujitsu, Yuichi Kusano - alternate
Neil Patterson (chair) - IBM
Ed Bratt - Oracle, Melissa Jacobus - alternate
Dominika Tasarz - Payara, Jadon Ortlepp - alternate
Jonathan Gallimore - Tomitribe, Cesar Hernandez - alternate
Eric (QingYu) Meng - Primeton - Elected Enterprise Representative
**Reza Rehman - Microsoft - Elected Enterprise Representative, Ed Burns - alternate**
Otavio Santana - Elected Committer Member Representative

Also please update 
**https://jakarta.ee/committees/specification/**, Committee Members list with (the change is in bold text here; on the website no need to need to bold)

Kenji Kazumura - Fujitsu
Tom Watson - IBM, Emily Jiang - alternate
Ed Bratt - Oracle, Dmitry Kornilov - alternate
Andrew Pielage - Payara, Petr Aubrecht - alternate
David Blevins - Tomitribe, Jean-Louis Monterio - alternate
Ivar Grimstad - PMC Representative
Marcelo Ancelmo - London Java Community - Elected Participant Member Representative, Martijn Verburg - alternate
Werner Keil - Elected Committer Member Representative
**Scott Stark - Red Hat- Elected Enterprise Representative, Scott Marlow - alternate**
Zhai Luchao - Shandong Cvicse Middleware Co. - Elected Enterprise Representative
Paul Buck - Eclipse Foundation (serves as chair, but is not a voting committee member)

Also please update 
**https://jakarta.ee/committees/steering/**, Committee Members list with (the change is in bold text here; on the website no need to need to bold)

Will Lyons (chair)- Oracle, Ed Bratt - alternate
Kenji Kazumura - Fujitsu, Yuichi Kusano - alternate
Ian Robinson - IBM, Alasdair Nottingham - alternate
Steve Millidge - Payara, Hadar Vorenshtein - alternate
David Blevins - Tomitribe, Cesar Hernandez - alternate
Martijn Verburg - London Java Community - Elected Participant Member Representative, Marcelo Ancelmo - alternate
Arjan Tijms - Elected Committer Member Representative
Dr. Jun Qian - Primeton - Elected Enterprise Representative
**Reza Rahman - Microsoft - Elected Enterprise Representative, Ed Burns - alternate**

### PR DESCRIPTION
**Pull request by oliviergoulet5**
Resolves #1556

### CODE REVIEW

**Comment by chrisguindon** [comment]
I've reach out to Netlify support to report that our Netlify previews are no longer working today when it was yesterday: https://github.com/jakartaee/jakarta.ee/pull/1555

File: content/committees/specification/_index.md [hunk 1]
```diff
@@ -17,8 +17,8 @@ The Specification Committee is responsible for implementing the ​[Jakarta EE S
 * Ivar Grimstad - PMC Representative
 * Marcelo Ancelmo - London Java Community - Elected Participant Member Representative, Martijn Verburg - alternate
```
Comment (ivargrimstad) [inline content/committees/specification/_index.md:18]: I don't think Martijn is the alternate, but check with @TanjaObradovic

File: content/committees/specification/_index.zh.md [hunk 2]
```diff
@@ -17,8 +17,8 @@ date: 2020-02-19T15:00:25-08:00
 * Ivar Grimstad - PMC Representative
 * Marcelo Ancelmo - London Java Community - Elected Participant Member Representative, Martijn Verburg - alternate
```
Comment (ivargrimstad) [inline content/committees/specification/_index.zh.md:18]: Same comment as for the English version

**Review by ivargrimstad — CHANGES_REQUESTED** [review]
Replace Martijn Verburg with  Marcin Kruglik from LJC as Participant Member rep in steering committee. No alternates specified yet

File: content/committees/specification/_index.md (hunk 1 above)
Comment (TanjaObradovic) [reply content/committees/specification/_index.md:18]: For the time being please keep Martijn.

**Comment by oliviergoulet5** [comment]
@ivargrimstad Updated to include Marcin Kruglik to the steering committee. Tanja provided Martjin Verburg as the alternate.

**Review by ivargrimstad — APPROVED** [review]

**Pull request merged**